src/state/slices/config_slice.ts

```diff
@@ -65,3 +79,49 @@ export const createConfigSlice: StateCreator<ConfigSlice, [], [], ConfigSlice> =
   setOutlineColor: (outlineColor) => set({ outlineColor }),
   setOpenTab: (openTab) => set({ openTab }),
 });
+
+export const serializeConfigSlice = (slice: ConfigSlice): SerializedStoreData => {
+  const ret: SerializedStoreData = {};
+  ret[UrlParam.SHOW_PATH] = encodeBoolean(slice.showTrackPath);
+  ret[UrlParam.SHOW_SCALEBAR] = encodeBoolean(slice.showScaleBar);
+  ret[UrlParam.SHOW_TIMESTAMP] = encodeBoolean(slice.showTimestamp);
+  // Export settings are currently not serialized.
+  ret[UrlParam.FILTERED_COLOR] = encodeColor(slice.outOfRangeDrawSettings.color);
+  ret[UrlParam.FILTERED_MODE] = slice.outOfRangeDrawSettings.mode.toString();
+  ret[UrlParam.OUTLIER_COLOR] = encodeColor(slice.outlierDrawSettings.color);
+  ret[UrlParam.OUTLIER_MODE] = slice.outlierDrawSettings.mode.toString();
+  ret[UrlParam.OUTLINE_COLOR] = encodeColor(slice.outlineColor);
+
+  ret[UrlParam.OPEN_TAB] = slice.openTab;
+  return ret;
+};
```

Comment:
curious: why not one big object literal?

```typescript
return {
    [UrlParam.SHOW_PATH]: encodeBoolean(slice.showTrackPath),
    [UrlParam.SHOW_SCALEBAR]: encodeBoolean(slice.showScaleBar),
    // ...etc
};
```

Reply:
No particular reason, I think I did this in case there were more complex serialization logic (like color ramp slice does, where it will sometimes not export certain keys). I've updated these to be an object literal.